pkg/ovs/ovsctl/appctl.go

```diff
@@ -59,11 +59,13 @@ type ovsCtlClient struct {
 	bridge string
 	// To allow injection for testing.
 	runAppCtl func(cmd string, needsBridge bool, args ...string) ([]byte, *ExecError)
+	runOVSCtl func(cmd string, args ...string) ([]byte, error)
```

Comment:
not because of your change, but there seems to be naming issues in this file. The file is called `appctl`, and the object is called `ovsCtlClient`.